CHANGELOG.md

```diff
@@ -1,13 +1,43 @@
 Changes by Version
 ==================
 
-1.8.0 (unreleased)
+1.8.0 (2018-11-12)
 ------------------
 
 #### Backend Changes
```

Comment:
I think we should split new features & bug fixes, right now it's a mixed bag

Reply:
done I have also unified the titles :scissors: